experimental/sdk/webpubsub-socketio-extension/package.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "socketio-webpubsub-extension",
+  "version": "1.0.0",
+  "socketio-version": "4.6.1",
+  "description": "Enable Socket.IO server support Azure Web PubSub Service",
+  "main": "./dist/src/index.js",
+  "types": "./dist/src/index.d.ts",
+  "author": "Microsoft",
+  "private": true,
+  "license": "MIT",
+  "directories": {
+    "lib": "src",
+    "example": "samples",
+    "test": "test"
+  },
+  "exports": {
+    "types": "./dist/src/index.d.ts"
+  },
+  "dependencies": {
```

Comment:
Does all these dependencies necessary, e.g `@socket.io/admin-ui`, `engine.io-client`, etc? We should minimize the dependencies.

Reply:
updated